### ISSUE TITLE
Configuration/Authentication Issue 7.0.1

### ISSUE THREAD

**Issue by PaulAndrewsYEG** [Brandawg93/homebridge-nest-cam#399]
Installed the most recent version 7.0.1 of the homebridge-nest-cam plugin and unable to get the plugin to generate the token. I have gone to settings and entered my username and password for my Google Account. I have authorized the login request through the Gmail app. Immediately get:

Could not get code. Error: Error: Request failed with status code 400
Could not generate refresh token.

I have cleared the Homebridge Accessory Cache
Removed the Nest Cam Plugin and configurations.

Attempted manual configuration per instructions. No change.

**Debug Output**
[1/29/2021, 10:47:16 AM] [Homebridge UI] [homebridge-nest-cam] Incoming Request: /login
[1/29/2021, 10:47:22 AM] [Homebridge UI] [homebridge-nest-cam] Incoming Request: /username
[1/29/2021, 10:47:25 AM] [Homebridge UI] [homebridge-nest-cam] Incoming Request: /password
**[1/29/2021, 10:47:34 AM] [Homebridge UI] [homebridge-nest-cam] Could not get code. Error: Error: Request failed with status code 400**
[1/29/2021, 11:16:01 AM] [Homebridge UI] [homebridge-nest-cam] Incoming Request: /generateToken
[1/29/2021, 11:17:13 AM] [Homebridge UI] [homebridge-nest-cam] Incoming Request: /getRefreshToken
[1/29/2021, 11:17:14 AM] [Homebridge UI] [homebridge-nest-cam] Invalid request url

Please answer these questions before submitting this issue:
```
On what device are you running homebridge: Synology NAS using Docker
On what iOS version are you having issues:  14.4
What camera types are having issues:  Nest Hello Doorbell Cam
Does this issue occur consistently or randomly:  Yes
If applicable, what was the last version that worked properly: 6.1.2
```

**Comment by Brandawg93** [comment]
Please try the manual login method.

**Comment by PaulAndrewsYEG** [comment]
As mentioned I tried the manual method. When I click on the Nest login page it asks me to choose an account, then grant Nest access to my Google Account which I have done multiple times now. From there it returns to Google.com and I open the Developer Tools. 

I copy the Nest login again and paste it into the window with the developer tools and grant access again. Then I click on the nest-account.... and it opens a new window with a URL which I copy and paste plug-in and click Log In. 

All that said, that seems to have done the trick. Not sure why it did not work the first three times I tried it,
